test/ode/ode_low_storage_rk_tests.jl

```diff
@@ -196,7 +195,7 @@ for prob in test_problems_nonlinear
   @test sim.𝒪est[:final] ≈ OrdinaryDiffEq.alg_order(alg) atol=testTol
 end
 integ = init(prob_ode_large, alg, dt=1.e-2, save_start=false, save_end=false, save_everystep=false)
-@test Base.summarysize(integ) ÷ Base.summarysize(u0_large) <= 5
+@test Base.summarysize(integ) ÷ Base.summarysize(u0_large) <= 4
```

Comment:
what's the theoretical size that these methods should be at?

Reply:
The theoretically best size would be `2`. We have two additional arrays since we store `u0` as part of the problem and we don't support function evaluations of the form `u2 = a*u2 + b*dt*f(u1)` using only two arrays.

Reply:
Got it. For the first, we should add a common interface argument for whether or not to re-use that `u0` as `u`. Defaulting to false, but this is a case where you'd usually want true. For the second part, I am inclined to say we just add that form to ODEFunction.

Reply:
That sounds good. A keyword such as `alies_u0`?

Reply:
yup that's a good name